SUPPOSE THE IQ IN A POPULATION IS NORMALLY DISTRIBUTED WITH MEAN 'MEW' = 100 WITH A STANDARD DEVIATION OF 15
RESEARCHERS WANT TO TEST A NEW MEDICATION TO SEE IF IT HAS AN EFFECT ON INTELLIGENCE. 
A SAMPLE POPULATION SIZE OF 20 PEOPLE TOOK THE MEDICATION HAVING IQS OF 140.
DOES THE MEDICATION AFFECT THE IQ?
if aplha /significance value = 0.05


In [1]:
#installed scipy and statsmodels libraries first
from scipy import stats
from statsmodels.stats.weightstats import ztest as ztest
data=[88, 92, 94, 94, 96, 97, 97, 97, 99, 99, 105, 109, 109, 109, 110, 112, 112, 113, 114, 115]
ztest(data, value=100)

(1.5976240527147705, 0.11012667014384257)

**As you observed that the result has 
tstat : 1.5976240527147705,
pvalue :  0.11012667014384257 which is not less than 0.05 we reject the null hypothesis**

if pvalue < significance value 'alpha', we reject the null hypothesis